File: oussema_zribi/TunGotTalent/src/tungottalent/Entite/Annonce.java

package tungottalent.Entite;


public class Annonce {
    private int idAnnonce;
    private User user;
    private String Nom;
    private String Description ;
    private double Prix;
    private String Etat;

    public Annonce(User user, String Nom, String Description, double Prix, String Etat) {
        this.user = user;
        this.Nom = Nom;
        this.Description = Description;
        this.Prix = Prix;
        this.Etat = Etat;
    }

    public Annonce(int idAnnonce, User user, String Nom, String Description, double Prix, String Etat) {
        this.idAnnonce = idAnnonce;
        this.user = user;
        this.Nom = Nom;
        this.Description = Description;
        this.Prix = Prix;
        this.Etat = Etat;
    }

    
    
    @Override
    public String toString() {
        return "Annonce{" + "idAnnonce=" + idAnnonce + ", user=" + user + ", Nom=" + Nom + ", Description=" + Description + ", Prix=" + Prix + ", Etat=" + Etat + '}';
    }

    
    public int getIdAnnonce() {
        return idAnnonce;
    }

    public User getUser() {
        return user;
    }

    public String getNom() {
        return Nom;
    }

    public String getDescription() {
        return Description;
    }

    public double getPrix() {
        return Prix;
    }

    public String getEtat() {
        return Etat;
    }

    public void setIdAnnonce(int idAnnonce) {
        this.idAnnonce = idAnnonce;
    }

    public void setUser(User user) {
        this.user = user;
    }

    public void setNom(String Nom) {
        this.Nom = Nom;
    }

    public void setDescription(String Description) {
        this.Description = Description;
    }

    public void setPrix(double Prix) {
        this.Prix = Prix;
    }

    public void setEtat(String Etat) {
        this.Etat = Etat;
    }
    
    
    

  }

File: oussema_zribi/Web/Fos/src/AnnonceBundle/Entity/Annonce.php
<?php

namespace AnnonceBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Annonce
 *
 * @ORM\Table(name="annonce")
 * @ORM\Entity(repositoryClass="AnnonceBundle\Repository\AnnonceRepository")
 */
class Annonce
{

    // ...

    /**
     * @ORM\ManyToOne(targetEntity="Categorie", inversedBy="Annonce")
     * @ORM\JoinColumn(name="category_id", referencedColumnName="id")
     */
    private $categorie;

    /**
     * @return \DateTime
     */
    public function getDateCreation()
    {
        return $this->dateCreation;
    }

    /**
     * @param \DateTime $dateCreation
     */
    public function setDateCreation($dateCreation)
    {
        $this->dateCreation = $dateCreation;
    }


    /**
     * @var \DateTime
     *
     * @ORM\Column(name="Date_creation", type="datetime", nullable=true)
     */
    private $dateCreation;

    /**
     * @var int
     *
     * @ORM\Column(name="idAnnonce", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $idAnnonce;

    /**
     * @var string
     *
     * @ORM\Column(name="Nom", type="string", length=255)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="Description", type="string", length=255)
     */
    private $description;

    /**
     * @var int
     *
     * @ORM\Column(name="Prix", type="integer")
     */
    private $prix;

    /**
     * @var string
     *
     * @ORM\Column(name="Etat", type="string", length=255)
     */
    private $etat;

    /**
     * @var string
     *
     * @ORM\Column(name="images", type="string", length=255, nullable=true)
     */
    private $images;

    /**
     * @return mixed
     */
    public function getUser()
    {
        return $this->user;
    }

    /**
     * @param mixed $user
     */
    public function setUser($user)
    {
        $this->user = $user;
    }


    /**
     * @ORM\ManyToOne(targetEntity="\FOSBundle\Entity\User")
     */
    private $user;


    /**
     * Get idAnnonce
     *
     * @return int
     */
    public function getIdAnnonce()
    {
        return $this->idAnnonce;
    }

    /**
     * @return mixed
     */
    public function getCategorie()
    {
        return $this->categorie;
    }

    /**
     * @param mixed $categorie
     */
    public function setCategorie($categorie)
    {
        $this->categorie = $categorie;
    }

    /**
     * Set nom
     *
     * @param string $nom
     *
     * @return Annonce
     */
    public function setNom($nom)
    {
        $this->nom = $nom;

        return $this;
    }

    /**
     * Get nom
     *
     * @return string
     */
    public function getNom()
    {
        return $this->nom;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return Annonce
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * Set prix
     *
     * @param integer $prix
     *
     * @return Annonce
     */
    public function setPrix($prix)
    {
        $this->prix = $prix;

        return $this;
    }

    /**
     * Get prix
     *
     * @return int
     */
    public function getPrix()
    {
        return $this->prix;
    }

    /**
     * Set etat
     *
     * @param string $etat
     *
     * @return Annonce
     */
    public function setEtat($etat)
    {
        $this->etat = $etat;

        return $this;
    }

    /**
     * Get etat
     *
     * @return string
     */
    public function getEtat()
    {
        return $this->etat;
    }

    /**
     * Set images
     *
     * @param string $images
     *
     * @return Annonce
     */
    public function setImages($images)
    {
        $this->images = $images;

        return $this;
    }

    /**
     * Get images
     *
     * @return string
     */
    public function getImages()
    {
        return $this->images;
    }
}

